### PR TITLE
Add Custom Hashtag to Event

### PR DESCRIPTION
**Pull request by popnbrown**
Created a new field to allow events to have custom hashtags.

### CODE REVIEW

File: models/event.py [hunk 1]
```diff
@@ -269,6 +270,16 @@ def gameday_url(self):
         else:
             return None
 
+    @property
+    def hashtag_event(self):
```
Comment (gregmarra) [inline models/event.py]: Is there a reason to not name this just `hashtag`?

**Comment by gregmarra** [comment]
I'd prefer the property to just be `hashtag`, but otherwise LGTM. @fangeugene any concerns?

**Comment by phil-lopreiato** [comment]
Do you think it would be worth having a way for users to suggest hashtags? That way, you and Eugene don't get pestered with requests to change them

**Comment by popnbrown** [comment]
@gregmarra I was just following the "facebook_event_url" function, thought that there was a reason for having event in the function name. Easy change, I'll get it up in a bit.

phil - That may be a slippery slope, unlike webcasts, hashtags aren't something a moderator can verify to be correct or not. Generally, they will come from the event organizing committee, and probably from very few. I don't want to put moderators in a situation where they approve an incorrect hashtag. There is a decent chance I'm overthinking this as I've been talking about hashtags with people for the last week.

**Comment by popnbrown** [comment]
@gregmarra committed, will travis re-run for the new commit?

**Comment by phil-lopreiato** [comment]
@popnbrown that's a good point - it probably doesn't matter much in the long run.

And yes, travis will rerun automatically for all new commits on PRs. You can see the output of the build [here](https://travis-ci.org/the-blue-alliance/the-blue-alliance/builds/55179671)

**Comment by synth3tk** [comment]
You can also see the status of the latest run at the bottom of this page (on desktop).

**Comment by popnbrown** [comment]
@gregmarra @fangeugene any chance of this moving live by next week?

**Comment by fangeugene** [comment]
Just checked out this PR. Looks good to me. @gregmarra any other comments?

**Comment by gregmarra** [comment]
I am good w this. Let's land it.

**Comment by popnbrown** [comment]
thanks gents!

**Pull request merged**